### ISSUE TITLE
Docker hub repo image does not work properly when deployed to Azure web app for containers.

### ISSUE THREAD

**Issue by Shubham-Isol** [jgraph/docker-drawio#35]
Hi,
We have been trying to deploy the docker hub image (jgraph/drawio:13.0.4) to Azure "Web App for Containers" service.
We can configure the service with docker hub correctly, but it is not picking the application settings from Azure.
We are trying to integrate the Onedrive using our own onedrive application.

Steps I followed:
1. Created a resource from Azure "Web App for Containers".
2. Configured the resource to connect with Docker-Hub respository with URL jgraph/drawio:13.0.4
3. Added the envrionment variables (DRAWIO_MSGRAPH_CLIENT_ID and DRAWIO_MSGRAPH_CLIENT_SECRET) into application settings in Azure portal - web app - Configuration.
4. When trying to access the web app from the deployed URL, it loads the diagrams frontend correctly, but the onedrive client_id is not updated, its still drawio's client_id.
5. So the authorization fails because the redirectUri is different.

So, Is there any specific configuration which we are missing while deploying the docker-hub container to azure web app.

**Comment by davidjgraph** [comment]
That's pretty old, try the latest version.

**Comment by Shubham-Isol** [comment]
yes tried the latest version too, but it does not load the application settings/environment variables from Azure.
Also we have a dependency on version 13.0.4

**Comment by davidjgraph** [comment]
OK, I've never used this Azure service, maybe someone else who has can answer.

**Comment by Kofl** [comment]
Same issue for me, if I look at the log stream output, it seems that Azure "Web App for Containers" does not pass the information to the docker command

INFO  - docker run -d -p 3904:8080 --name webapp-drawio-docker_8_9bb02804 -e WEBSITE_AUTH_ENABLED=False -e WEBSITES_ENABLE_APP_SERVICE_STORAGE=false -e WEBSITE_SITE_NAME=xyz -e PORT=8080 -e WEBSITE_ROLE_INSTANCE_ID=0 -e WEBSITE_HOSTNAME=webapp-drawio-docker.azurewebsites.net -e WEBSITE_INSTANCE_ID=xxx -e WEBSITE_USE_DIAGNOSTIC_SERVER=False jgraph/drawio
